store/generated_key/eigenda/eigenda.go

```diff
@@ -98,29 +99,30 @@ func (e Store) Put(ctx context.Context, value []byte) ([]byte, error) {
 		func() (*disperser.BlobInfo, error) {
 			return e.client.PutBlob(ctx, value)
 		},
-		retry.RetryIf(func(err error) bool {
-			st, isGRPCError := status.FromError(err)
-			if !isGRPCError {
-				// api.ErrorFailover is returned, so we should retry
-				return true
-			}
-			//nolint:exhaustive // we only care about a few grpc error codes
-			switch st.Code() {
-			case codes.InvalidArgument:
-				// we don't retry 400 errors because there is no point,
-				// we are passing invalid data
-				return false
-			case codes.ResourceExhausted:
-				// we retry on 429s because *can* mean we are being rate limited
-				// we sleep 1 second... very arbitrarily, because we don't have more info.
-				// grpc error itself should return a backoff time,
-				// see https://github.com/Layr-Labs/eigenda/issues/845 for more details
-				time.Sleep(1 * time.Second)
-				return true
-			default:
-				return true
-			}
-		}),
+		retry.RetryIf(
+			func(err error) bool {
+				st, isGRPCError := status.FromError(err)
+				if !isGRPCError {
+					// api.ErrorFailover is returned, so we should retry
+					return true
+				}
```

Comment:
more IDE formatting changes. would be nice to have a standardized (maybe deterministic) formatting rules. Is it possible to have your IDE follow the rules we have in the makefile?

Reply:
reverted

[722a9fc](https://github.com/Layr-Labs/eigenda-proxy/pull/340/commits/722a9fcacf9bb0d2b43ec33d953d76445d9127c6)